### PR TITLE
ResearchKit: Support 'disableDisagree', 'hideDefaultHeader' options on ConsentReviewStep

### PR DESCRIPTION
**Pull request by overcyn**
Fixes https://github.com/CareEvolution/RKStudio-Participant/issues/984
Related CEVResearchKit change https://github.com/CareEvolution/CEVResearchKit/pull/257

Adds flags to hide the disagree button and to hide the default header.

### ResearchKit 1
Screen Shot 2021-08-10 at 4 29 38 PM

### ResearchKit 2
Screen Shot 2021-08-10 at 4 27 27 PM

# Testing notes

1. Create a survey with the following JSON https://gist.githubusercontent.com/overcyn/8c5b2c2205454528a00f85dd0212db3d/raw/2394638508b588de3a004f871321e2b592c545a3/gistfile1.txt
2. Open the survey and navigate through
3. Note that the disagree button is hidden and the Review header is missing.

### CODE REVIEW

File: ORK1Kit/ORK1Kit/Consent/ORK1ConsentReviewStepViewController.m [hunk 1]
```diff
@@ -219,9 +219,9 @@ - (ORK1ConsentReviewController *)makeDocumentReviewViewController {
     }
     
     NSString *html = [document mobileHTMLWithTitle:ORK1LocalizedString(@"CONSENT_REVIEW_TITLE", nil)
```
Comment (mmertsock) [inline ORK1Kit/ORK1Kit/Consent/ORK1ConsentReviewStepViewController.m]: Should the `CONSENT_REVIEW_TITLE` title text also be hidden when hideDefaultHeader is true? My (uncertain) understanding of "just lets the consent webview stretch to the top of the page" in https://github.com/CareEvolution/Consumers/issues/9957 is that the title should also be hidden.

Comment (chrisnowak) [reply ORK1Kit/ORK1Kit/Consent/ORK1ConsentReviewStepViewController.m]: Yes...the "Review" title should be hidden...the content content should just stretch to the top

Comment (overcyn) [reply ORK1Kit/ORK1Kit/Consent/ORK1ConsentReviewStepViewController.m]: Fixed. Updated screenshots.

**Review by mmertsock — APPROVED** [review]
LGTM 🚢

**Pull request merged**